### PR TITLE
Fix the populate_reservations.py script

### PR DESCRIPTION
**Pull request by Index01**
popres
    - samples/exampleGuest for guests
    - samples/exampleRoom for rooms
    - based on 2022 dataset

### CODE REVIEW

File: samples/exampleMainRoomList.csv [hunk 1]
```diff
@@ -0,0 +1,239 @@
+Floor,Room,Room Type,Connected To Room,First Name,Last Name,Check-in Date,Check-out Date,Art Room,Art Name / Placed Name,Placed By,Changeable,Change Reason,Notes,Paying guest?,Ticket ID in SecretParty,Placement Team Notes,,ROOMBAHT
```
Comment (otakup0pe) [inline samples/exampleMainRoomList.csv:1]: Flagging for future us that we should have some sample data that includes more fields - at least fields that we plan on bubbling up in the gooey.

**Review by otakup0pe — CHANGES_REQUESTED** [review]
Let's get #26 merged in and then rebase this PR off of it before merging it otherwise lgtm ✨

File: samples/exampleMainRoomList.csv (hunk 1 above)
Comment (Index01) [reply samples/exampleMainRoomList.csv:1]: sounds good! 
ja the dataset i had contains peoples names and emails, from last year. so I filtered for only the rooms marked for roombot before adding them to the csv(which is why they all end in `,R`). Would be good to get a copy of the complete spreadsheet from this yr.

**Comment by otakup0pe** [comment]
Oh / And / Also the script seems to run locally 🌞

File: samples/exampleMainRoomList.csv (hunk 1 above)
Comment (mpesavento) [reply samples/exampleMainRoomList.csv:1]: I'm behind on my task of updating the schema, that will help clarify

**Comment by Index01** [comment]
I think we want to close this MR. The changes to pop_res.py are superceded by @otakup0pe changes and the .csv changes I incorporated into my refactor MR #27

**Comment by mpesavento** [comment]
Closing this PR, as most of the work was done elsewhere in https://github.com/Index01/RoomBot/pull/26

**Pull request closed without merging**